src/sid/simulate.py

```diff
@@ -83,13 +83,13 @@ def get_simulate_func(
             what you need to calculate moments to make the simulation and calculation
             of moments faster. The categories are "initial_states", "disease_states",
             "testing_states", "countdowns", "contacts", "countdown_draws", "group_codes"
-            "infection_reason" and "other".
+            and "other".
         optional_state_columns (dict): Dictionary with categories of state columns
             that can additionally be added to the states DataFrame, either for use in
             contact models and policies or to be saved. Most types of columns are added
             by default, but some of them are costly to add and thus only added when
             needed. Columns that are not in the state but specified in ``saved_columns``
-            will not be saved. The categories are "contacts" and "reason_for_infection".
+            will not be saved. The sole category is currently "contacts".
```

Comment:
I think this is outdated and there are more columns. For example in `testing_demand.py` you use:

![image](https://user-images.githubusercontent.com/9536380/100626110-acbff580-3325-11eb-9f99-bb24fe2d3df6.png)

Reply:
Even though I came up with this concept I think I would get rid of the `optional_state_columns` and instead keep a list (in `config.py`) of columns that are only generated if they are in the `saved_columns`. It is just not an intuitive interface.

Reply:
Postponed and planned in #54.